config/manager/manager.yaml

```diff
@@ -74,6 +74,7 @@ spec:
           - name: CLUSTER_NAME
             value: CLUSTER_NAME_VALUE # Replace it with the cluster name where numaplane will get deployed.
         image: numaplane-controller:latest
+        imagePullPolicy: IfNotPresent
```

Comment:
Why do you adding this configuration? I think it is good to pull each time to avoid local image is out dated.